blockchain/blockchain-wallet-v4-frontend
# feat/single-socket

## Description 
Updating socket middleware to the new api `wss://ws.staging.blockchain.info/coins`.
A single socket now handles btc, bch, and eth.

## Testing Steps 
1. Change staging config `WEB_SOCKET_URL` to `wss://ws.staging.blockchain.info/coins`
2. Use two tabs to send btc, bch, or eth funds. Notice alert updates when transactions are received.
3. View transactions page while receiving funds (`/btc/transactions`, `/bch/transactions`, `/eth/transactions`). Notice that transaction appear without having to refresh.
